app/controllers/studios_controller.rb

```diff
@@ -0,0 +1,29 @@
+class StudiosController < ApplicationController
+  def index
+    @studios = Studio.all
+  end
+
+  def show
+    @studio = Studio.find(params[:id])
+  end
+
+  def new
+    @studio = Studio.new
+  end
+
+  def create
+    @studio = Studio.new(studio_params)
+    @studio.user = current_user
+    if @studio.save
+      redirect_to "/studios/#{@studio.id}"
```

Comment:
why not studio_path(@studio) ? This seems to be the accepted convention

https://edgeapi.rubyonrails.org/classes/ActionController/Redirecting.html

and in the lectures, too.

https://kitt.lewagon.com/camps/795/lectures/05-Rails%2F02-Rails-CRUD#source

Reply:
Fixed (commit: 26aa78d)